jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/timstorage/Anchor.java

```diff
@@ -1,57 +1,62 @@
 /*******************************************************************************
- * Copyright 2018 572682
- * 
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * Copyright 2018 572682.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not
  * use this file except in compliance with the License.  You may obtain a copy
- * of the License at
- * 
- *   http://www.apache.org/licenses/LICENSE-2.0
- * 
- * Unless required by applicable law or agreed to in writing, software
+ * of the License at</p>
+ *
+ *   <p>http://www.apache.org/licenses/LICENSE-2.0</p>
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
  * License for the specific language governing permissions and limitations under
- * the License.
+ * the License.</p>
  ******************************************************************************/
+
 package us.dot.its.jpo.ode.plugin.j2735.timstorage;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
-
+import lombok.EqualsAndHashCode;
 import us.dot.its.jpo.ode.plugin.asn1.Asn1Object;
 
-@JsonPropertyOrder({ "lat", "long", "elevation" })
+/**
+ * Anchor point for a path.
+ */
+@JsonPropertyOrder({"lat", "long", "elevation"})
+@EqualsAndHashCode(callSuper = false)
```

Comment:
Could we also use @Data? Or does the json property name "llong" name mess with that?

Reply:
I think we can use `@Data` here. I've added this annotation and moved the `@JsonProperty` annotations to the field delcarations.